frontend/src/components/App/Settings/Settings.tsx

```diff
@@ -127,6 +104,83 @@ export default function Settings() {
           },
         ]}
       />
+      <Box
+        sx={{
+          mt: '2',
+          borderTop: '1px solid',
+          borderTopColor: 'divider',
+          pt: '2',
+        }}
+      >
+        <Box
+          sx={{
+            display: 'flex',
+            alignItems: 'baseline',
+            px: 1.5,
+            py: 1,
+          }}
+        >
+          <Typography
+            variant="body1"
+            sx={theme => ({
+              textAlign: 'left',
+              color: theme.palette.text.secondary,
+              fontSize: '1rem',
+              [theme.breakpoints.down('sm')]: {
+                fontSize: '1.5rem',
+                color: theme.palette.text.primary,
+              },
+            })}
+          >
+            {t('translation|Theme')}
+          </Typography>
+        </Box>
+        <Box
+          sx={{
+            width: '100%',
+            margin: 'auto',
+            pb: 5,
+          }}
+        >
+          <Box
+            sx={{
+              display: 'grid',
+              gridTemplateColumns: 'repeat(auto-fit, minmax(130px, 1fr))',
+              gap: 2,
+              justifyContent: 'center',
+            }}
+          >
+            {appThemes.map(it => (
+              <Box
+                key={it.name}
+                role="button"
+                tabIndex={0}
+                onKeyDown={e => {
+                  if (e.key === 'Enter' || e.key === ' ') dispatch(setTheme(it.name));
+                }}
+                sx={{
+                  cursor: 'pointer',
+                  border: themeName === it.name ? '3px solid' : '1px solid',
```

Comment:
for "active" state let's use a primary color (theme.palette.primary) for the 1px border so it provides more contrast. In the current version it is a bit hard to see which item is selected.